### DISCUSSION TITLE
I would like to inquire about multi-agent.

### DISCUSSION THREAD

**Discussion by luxdolorosa** [microsoft/autogen#3941] [Q&A]
Hello. I am learning autogen and want to use it in my work.

Our team has quite a bit of disagreement about the term multi-agent.

I have a question about the configuration of Chat with OpenAI Assistant using Function Call - View Notebook under Multi-Agent Chat with OpenAI Assistants in the loop.

A configuration where one agent (oss_analyst in the code) calls the tool does not seem to be a multi-agent configuration.

Is there a reason why this is classified as multi-agent?

**Comment by ekzhu** [comment]
It's not multi agent if you only use a single agent. Bad heading, should be called "OpenAI assistant agent"
